guests/reth-ethereum/Cargo.toml

```
@@ -5,6 +5,16 @@ edition = "2021"
 
 [workspace]
 
+[profile.release]
+codegen-units = 1
+debug = 1
+lto = "fat"
+opt-level = 3
```

opt-level and debug is already included in the default release profile and won't have any effect.

```suggestion
codegen-units = 1
lto = "fat"
```

---

While opt-level is already 3 in release, the default for debug is 0:

https://doc.rust-lang.org/cargo/reference/profiles.html#release